src/internal/connector/graph/http_wrapper_test.go

```diff
@@ -0,0 +1,45 @@
+package graph
+
+import (
+	"net/http"
+	"testing"
+
+	"github.com/alcionai/clues"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/alcionai/corso/src/internal/tester"
+)
+
+type HTTPWrapperIntgSuite struct {
+	tester.Suite
+}
+
+func TestHTTPWrapperIntgSuite(t *testing.T) {
+	suite.Run(t, &HTTPWrapperIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tester.M365AcctCredEnvs}),
+	})
+}
+
+func (suite *HTTPWrapperIntgSuite) TestNewHTTPWrapper() {
+	ctx, flush := tester.NewContext()
+	defer flush()
+
+	var (
+		t  = suite.T()
+		hw = NewHTTPWrapper()
+	)
+
+	resp, err := hw.Request(
```

Comment:
Should we mock it with `gock`? There is no real need to actually make the request, right?

Reply:
I actually do want it to make the request, in this case.  Thus, integration rather than unit suite.  Gives me  more assurance that the client actually functions end-to-end.